src/OpenSpaceToolkit/Astrodynamics/Estimator/TLESolver.cpp

```diff
@@ -0,0 +1,307 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/Utility.hpp>
+
+#include <OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Estimator/TLESolver.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Model/BrouwerLyddaneMean/BrouwerLyddaneMeanLong.hpp>
+
+namespace ostk
+{
+namespace astrodynamics
+{
+namespace estimator
+{
+
+using ostk::core::type::Shared;
+
+using ostk::mathematics::object::VectorXd;
+
+using ostk::physics::coordinate::Frame;
+using ostk::physics::coordinate::Position;
+using ostk::physics::coordinate::Velocity;
+using ostk::physics::environment::gravitational::Earth;
+using ostk::physics::time::Instant;
+
+using ostk::astrodynamics::trajectory::orbit::model::blm::BrouwerLyddaneMeanLong;
+using ostk::astrodynamics::trajectory::orbit::model::SGP4;
+
+TLESolver::Analysis::Analysis(const TLE& aDeterminedTLE, const LeastSquaresSolver::Analysis& anAnalysis)
+    : determinedTLE(aDeterminedTLE),
+      solverAnalysis(anAnalysis)
```

Comment:
Same, I feel that either we expand all the fields to a native TLESolver Analysis, or we expand none of the fields and just use the LSSolver Analysis

Reply:
Same as the other comment, since a TLE is it's own object I think it's convenient having this for the user.